src/api/server.ts

```diff
@@ -1,6 +1,8 @@
 import axios from 'axios';
 
 const baseUrl = process.env.REACT_APP_BASE_URL;
+//^^ in case there's an api error, try changing this line to: const baseUrl = 'http://localhost:9000';
+// tip from Gordon
```

Comment:
**Ternary expression may help here**: {expr} ? {stmt1 if true} : {stmt2 if false}
Change line 3 to:
`const baseUrl = process.env.NODE_ENV === 'development' ? 'http://localhost:9000' : process.env.REACT_APP_BASE_URL`